src/hooks/useTransactions.ts

```diff
@@ -1,17 +1,22 @@
 import { useCallback } from 'react';
 import { useSelector } from 'react-redux';
 import { AppState } from '@rainbow-me/redux/store';
-import { ethereumUtils } from '@rainbow-me/utils';
+import { ethereumUtils, isLowerCaseMatch } from '@rainbow-me/utils';
 
 export default function useTransactions() {
-  const transactions = useSelector(({ data }: AppState) => data.transactions);
+  const pendingTransactions = useSelector(
+    ({ data }: AppState) => data.pendingTransactions
+  );
 
   const getTransactionByHash = useCallback(
```

Comment:
ideally would rename this hook to `usePendingTransactions` and the function to `getPendingTransactionByHash`